src/helm/desk/templates/secrets.yaml

```diff
@@ -22,3 +22,9 @@ stringData:
 {{- if .Values.mail_provisioning_api_credentials }}
   MAIL_PROVISIONING_API_CREDENTIALS: {{ .Values.mail_provisioning_api_credentials }}
 {{- end }}
+{{- if .Values.djangoEmailHostUser }}
+  DJANGO_EMAIL_HOST_USER: {{ .Values.djangoEmailHostUser }}
```

Comment:
maybe don't put this as a secret so we get the info without having to decode secrets. Since it's not secret...